src/uu/expand/src/expand.rs

```diff
@@ -58,14 +64,14 @@ struct Options {
 }
 
 impl Options {
-    fn new(matches: getopts::Matches) -> Options {
-        let tabstops = match matches.opt_str("t") {
+    fn new(matches: &ArgMatches) -> Options {
+        let tabstops = match matches.value_of("tabstops") {
+            Some(s) => tabstops_parse(s.to_string()),
             None => vec![DEFAULT_TABSTOP],
-            Some(s) => tabstops_parse(s),
         };
 
-        let iflag = matches.opt_present("i");
-        let uflag = !matches.opt_present("U");
+        let iflag = matches.is_present("iflag");
```

Comment:
Please follow the same declarations as in other for the options :)
For example:
https://github.com/uutils/coreutils/blob/517b2a6a2244637a5f5331565592873a734085c7/src/uu/chown/src/chown.rs#L30

Reply:
Thanks for the advice. cd775ed fix this.